examples/diagram-fit-variants/index.html

```diff
@@ -113,7 +113,7 @@ <h2>Various fit variants</h2>
                                 <a id="toggle-dropdown-diagram-choice" href="#" class="btn btn-primary dropdown-toggle" tabindex="0" style="background: black">
                                     Select a BPMN Diagram <span class="icon icon-caret icon-in-the-middle"></span>
                                 </a>
-                                <ul class="menu" id="dropdown-diagram-choice">
+                                <ul class="menu custom-dropdown" id="dropdown-diagram-choice">
```

Comment:
```suggestion
                                <ul class="custom-dropdown" id="dropdown-diagram-choice">
```

Reply:
@csouchet I cannot remove the menu class, otherwise, the dropdown elements are displayed like this

![image](https://user-images.githubusercontent.com/27200110/97695266-11c4c900-1aa4-11eb-8880-250c2727686b.png)